use_cases/init-process-for-fluent-bit/README.md

```diff
@@ -0,0 +1,80 @@
+
+```
+AWS_REGION / ECS_LAUNCH_TYPE / ECS_CLUSTER / ECS_FAMILY
+ECS_TASK_ARN / ECS_TASK_ID / ECS_REVISION / ECS_TASK_DEFINITION
+```
+
+You can use them as env vars directly in the Fluent Bit config.
+
+
+
+### How to use multi-config feature?
+
+1. Use `aws-for-fluent-bit` images with `init` tag.
+
+2. Specify config files as environment variable in the FireLens configuration Environment setting.
+
+   **Example:**
+
+   ```
+   "environment": [
+                   {
+                       "name": "aws_fluent_bit_init_s3_1",
+                       "value": "arn:aws:s3:::yourBucket/aaaaa.conf"
+                   },
+                   {
+                       "name": "aws_fluent_bit_init_s3_2",
+                       "value": "arn:aws:s3:::mybucket/bbbbb.conf"
+                   },
+                   {
+                       "name": "aws_fluent_bit_init_file_1",
+                       "value": "/ecs/ccccc.conf"
+                   }
+               ], 
+   ```
+
+   **We support two kinds of config files:**
+
+   * Config files which stored in your S3 bucket
+     You need to set the name of the env var using prefix `aws_fluent_bit_init_s3_`, the number after this prefix cannot be repeated, and set the `ARN` of your config file as the value of the env var.
+
+   * Config files which from [our built-in config folder](https://github.com/aws/aws-for-fluent-bit/tree/mainline/ecs)
+
+     You need to set the name of the env var using prefix `aws_fluent_bit_init_file_`, the number after this prefix cannot be repeated, and set the `Path` of your config file as the value of the env var.
+
+3. Set up your ECS Task role
+
+   If you specify the config file from your S3 bucket, the init process will download it from your bucket. So please add the following permissions: 
+
+   ``` 
+   {
+       "Version": "2012-10-17",
+       "Statement": [
+           {
+               "Effect": "Allow",
+               "Action": [
+                   "s3:GetObject",
+                   "s3:GetBucketLocation"
+               ],
+               "Resource": "*"
+           }
+       ]
+   }
+   ```
+
+
+
+### How init process works?
+
+1. The init process will request [ECS Task Metadata](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-metadata-endpoint-v4.html). Get valuable parts from the responses and set them using `export` as environment variables.
+2. After you set the ARN or Path of the config files as environment variables in the ECS Task Definition, the init process will collect all config files which you specified (download files which come from S3 if needed).
+3. The init process will process these config files one by one, use @INCLUDE keyword to add config files to the main config file. And the init process will check if each config file is a parser config. If it is a parser config, change the original command, add -R specify that parser.
+4. The init process will finally invoke Fluent Bit with the modified main configuration file.
```

Comment:
Its not clear here that it will @INCLUDE the FireLens generated config file. I think that an example of how this works would be good- I think you have stuff like htis in the design. Show an entire example- the user env input in the task def, the config files and their content, and then the main config file init will create.

Reply:
Yea, I will rewrite this part to make it more clear